app/components/typography/paragraph/styles.js

```diff
@@ -0,0 +1,15 @@
+import { StyleSheet } from "react-native";
+
+import theme from "../../../theme/theme";
+
+export default StyleSheet.create({
```

Comment:
may be we should return not a style but a styleCreator
```
export default ({mode, ...})=>{
    return StyleSheet.create({
         container:{
              backgroundColor:theme[mode+"Color"],
          }
     })
}
```
we could even wrap it with a higher order function to add optimisations like memoization.
It could allow us may be to create dynamique styleSheet -> dynamique theme

Reply:
use would be something like this
```
import styleSheetCreator from './styles'

...
render(){
    const styles = styleSheetCreator({mode: this.props.mode, size:this.props.size}) 
    return <MyComponent style={styles.toto} />
}
....

```

Reply:
let's compose all the things! I like this!